cluster/util/environment.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import (
+	"os"
+	"strings"
+)
+
+func EnvironmentVariablesToMap() map[string]string {
+	m := make(map[string]string, len(os.Environ()))
```

Comment:
hi,
as described in the previous PR, all configurations have to be done before the client is instantiated
and struct fields instead of map should be used

Reply:
Working on this right now. Writing a couple of tests and I'll push the changes.

Reply:
✂️